nifi-framework-bundle/nifi-framework-extensions/nifi-py4j-framework-bundle/nifi-python-framework/src/test/resources/python/framework/processor_with_dependencies/requirements.txt

```diff
@@ -0,0 +1 @@
+google-cloud-vision==3.7.4
```

Comment:
This file is missing the license header.

Reply:
I have added the header in 6d97284da4104a0695a86c23ecd80aa5ac104629